content/writing-pipelines/index.md

```diff
@@ -246,17 +397,24 @@ stage SORT_ITEMS(
 )
 ~~~~
 
-`mrf` is an "opinionated" formatter, inspired by tools like `gofmt`, therefore we will borrow
-[their explanation](https://blog.golang.org/go-fmt-your-code) of the benefits of canonical code formatting:
+`mrf` is an "opinionated" formatter, inspired by tools like `gofmt`, therefore
+we will borrow [their explanation](https://blog.golang.org/go-fmt-your-code) of
+the benefits of canonical code formatting:
 
 - **Easier to write**: never worry about minor formatting concerns while hacking away.
 - **Easier to read**: when all code looks the same you need not mentally convert others' formatting style into something you can understand.
 - **Easier to maintain**: mechanical changes to the source don't cause unrelated changes to the file's formatting; diffs show only the real changes.
 - **Uncontroversial**: never have a debate about spacing or brace position ever again!
 
-`mrf` takes a list of MRO filenames as arguments. By default, it will output the formatted code back to `stdout`. If given the `--rewrite` option, it will write the formatted code back into the original files. If given the `--all` option, it will rewrite all MRO files found in your `MROPATH`. For consistency of your MRO codebase, consider configuring editor save-hooks or git commit-hooks that run `mrf --rewrite` or `mrf --all`.
+`mrf` takes a list of MRO filenames as arguments. By default, it will output
+the formatted code back to `stdout`. If given the `--rewrite` option, it will
+write the formatted code back into the original files. If given the `--all`
+option, it will rewrite all MRO files found in your `MROPATH`. For consistency
+of your MRO codebase, consider configuring editor save-hooks or git
+commit-hooks that run `mrf --rewrite` or `mrf --all`.
 
-`mrf` does not support any arguments that affect the formatting, otherwise it would not be canonical!
+`mrf` does not support any arguments that affect the formatting, otherwise it
```

Comment:
Would be good to mention `--includes` here

Reply:
Done.